src/core_plugins/vega/public/_vega_vis.scss

```diff
@@ -0,0 +1,132 @@
+
+.vgaVis {
+  display: flex;
+  flex: 1 1 100%;
+  position: relative;
+  // flex-direction determined by js
+}
+
+.vgaVis__view {
+  z-index: 0;
+  flex: 1 1 100%;
+
+  display: block;
+  max-width: 100%;
+  max-height: 100%;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+
+  // BUG #23514: Make sure Vega doesn't display the controls in two places
+  .vega-bindings {
+    display: none !important;
+  }
+}
+
+.vgaVis__controls {
+  display: flex;
+  @include euiFontSizeS;
```

Comment:
mixin first.

Reply:
Well, it's actually updated in the codebase but GH is refusing to show that it's been changed...